src/modules/previewpane/MonacoPreviewHandler/index.html

```diff
@@ -39,24 +44,59 @@
 			/*Hides alert box */
 			display:none!important
 		} 
+
+		.context-menu { 
+			position: absolute; 
+		} 
+		.menu {
+			background-color: #fff;
+			border-radius: 10px;
+			box-shadow: 0 10px 20px rgb(64 64 64 / 5%);
+			width: 60px;
+		}
+		.menu > li > a {
+			font: inherit;
+			border: 0;
+			width: 100%;
+			align-items: center;
+			position: relative;
+			text-decoration: unset;
+			color: #000;
+			font-weight: 500;
+			transition: 0.1s linear;
+		}
+		.menu > li > a:hover {
+			background:#f1f3f7;
+			color: #4b00ff;	
+		}
+		.menu > li > a > i {
```

Comment:
`<i>` element has been deprecated for a while. Where is this used and is there a better way?
(W3C says to use css)

Reply:
It's a leftover for something I tried. Most of the css can get removed